### ISSUE TITLE
Duplicate reported issues

### ISSUE THREAD

**Issue by KamasamaK** [cflint/CFLint#642]
I've found more instances like #600

Given the code 
```cfml
<cfset variables.jsonObj = new Json()>

<cfset variables.foo = deserializeJSON(variables.jsonObj.serialize(variables.response))>
```

The rules `VAR_HAS_PREFIX_OR_POSTFIX` and `VAR_IS_TEMPORARY` are reported not only for the assignment but for all usages of `jsonObj`. I've only included one usage for simplicity. I think these should only trigger on variable declaration or assignment.
